kola/tests/rpmostree/status.go

```diff
@@ -57,9 +59,9 @@ type simplifiedRpmOstreeStatus struct {
 	Deployments []rpmOstreeDeployment
 }
 
-// getRpmOstreeStatusJSON returns an unmarshal'ed JSON object that contains
+// GetRpmOstreeStatusJSON returns an unmarshal'ed JSON object that contains
 // a limited representation of the output of `rpm-ostree status --json`
-func getRpmOstreeStatusJSON(c cluster.TestCluster, m platform.Machine) (simplifiedRpmOstreeStatus, error) {
+func GetRpmOstreeStatusJSON(c cluster.TestCluster, m platform.Machine) (simplifiedRpmOstreeStatus, error) {
```

Comment:
I think this function might fit in the [tests util folder](https://github.com/coreos/mantle/tree/master/kola/tests/util).

Reply:
OK, I'm going to break out this change into a separate PR.

Should the `structs` get defined in the `util` package too?

Reply:
Yeah, if the structs are primarily created by this function they should probably live there as well.

As for breaking it out into a new PR, it's fine to me either way.